### DISCUSSION TITLE
ignoreHttpsErrors not working with puppeteer >= 23

### DISCUSSION THREAD

**Discussion by Scantech-Val** [spatie/browsershot#877] [Bugs]
Puppeteer made breaking changes in v23, one of which is renaming ignoreHttpsErrors to acceptInsecureCerts (https://github.com/puppeteer/puppeteer/issues/12756). 

I am using a self signed certificate for https in a dev environment and I encounter a `Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID`, even if i try the renamed version of ignoreHttpsErrors. I tried to use the new option but to no success like so:

```
$pdf = Browsershot::html($content) // content has my html with https link to my css
                ->format('A4')
                ->showBrowserHeaderAndFooter()
                ->footerHtml($footer)
                ->setOption('args', ['--disable-web-security'])
                ->newHeadless()
                ->waitUntilNetworkIdle();

$pdf = $pdf->ignoreHttpsErrors()->noSandbox()->setOption('acceptInsecureCerts', true);
//var_dump($pdf->consoleMessages());

$pdf->save($output_path);
```

**Comment by clementmas** [comment]
Fixed in https://github.com/spatie/browsershot/pull/899
